checkout/src/main/java/com/checkout/validation/validator/CvvValidator.kt

```diff
@@ -41,9 +59,17 @@ internal class CvvValidator : Validator<CvvValidationRequest, Unit> {
                 ValidationError.CVV_CONTAINS_NON_DIGITS,
                 "CVV should contain only digits"
             )
+            isEager && cvv.length > cvvLength.last() -> throw ValidationError(
```

Comment:
I didn't get logic under `cvv.length > cvvLength.last()`.
`cvvLength` is an unsorted set. It can be `setOf(3, 4)` or `setOf(4, 3)`.